.github/workflows/builder.yml

```diff
@@ -11,6 +11,7 @@ on:
     - Dockerfile
     - build.yaml
     - 'rootfs/**'
+  workflow_dispatch:
```

Comment:
```suggestion
```
That is trigger on merge that generate also the version number, this would only overwrite the build before

Reply:
That was just a workaround to manually trigger the build in my dev branch because builder.yml just listens for changes on master branch of course.